app/scripts/modules/core/help/helpWidget.component.ts

```diff
@@ -0,0 +1,90 @@
+import * as moment from 'moment';
+import {Component, Input, OnChanges, OnDestroy, OnInit, SimpleChanges} from '@angular/core';
+
+interface IHelpFieldContents {
+  content: string;
+  placement: string;
+}
+
+@Component({
+  selector: 'deck-help-widget',
+  template: `
+    <ng-template #popoverContent>
+      <div [innerHTML]="contents.content" (mouseenter)="popoverHovered()" (mouseleave)="hidePopover(false)"></div>
+    </ng-template>
+    <div style="display: inline-block;">
+      <div ng-if="expand && contents.content"
+           class="help-contents small"
+           [popover]="popoverContent"></div>
+      <a href class="help-field" ng-if="!expand && contents.content"
```

Comment:
Doh!  That's a really good point on the `ng-if`, totally missed that copying the template from the original, ng1 component - I'm not sure why it's working, but it is...I'll change it and see what happens.

For `triggers` and `containers`, the values are strings (think `@` bindings in ng1 components, the attribute value is not evaluated against the controller's scope) whereas `[]=""` values are property bindings to be evaluated (think `<` or `=` bindings in ng1 components where the attribute value IS evaluated against the controller's scope).